apps/provisioning/pkg/apis/provisioning/v0alpha1/jobs.go

```diff
@@ -221,9 +221,11 @@ func (in JobStatus) ToSyncStatus(jobId string) SyncStatus {
 }
 
 type JobResourceSummary struct {
-	Group    string `json:"group,omitempty"`
-	Resource string `json:"resource,omitempty"`
-	Total    int64  `json:"total,omitempty"` // the count (if known)
+	Group        string `json:"group,omitempty"`
+	Kind         string `json:"kind,omitempty"`
+	SingularName string `json:"singularName,omitempty"`
+	Resource     string `json:"resource,omitempty"`
```

Comment:
🤔   this seems odd to require:
* Kind
* SinglurName
* Resource
for the summary... we need to pick one of them (Kind or Resource make sense to me)

whichever is easier to populate!

Reply:
Is `SinglurName` only used in UI? I think we could work around that on the FE instead then.